src/components/SideBarItem/SideBarItem.jsx

```diff
@@ -3,26 +3,23 @@ import PropTypes from 'prop-types';
 import { SideBarButton, SideBarIcon, Text } from './SideBarItemStyled';
 import Icons from '../Icons/Icons';
 
-function SideBarItem({
+const SideBarItem = ({
   onClick,
   isActive,
   alt,
   name,
-  fill,
   stroke,
   size,
-  children,
-}) {
+  children
+}) => {
   return (
     <SideBarButton
       onClick={() => onClick()}
-      style={{
-        color: isActive ? 'var(--color-brand-blue)' : 'var(--color-grey-9)',
-        fontWeight: isActive ? '700' : '400',
-      }}
+      color={isActive}
```

Comment:
Put the isActive prop down and use it to manipulate with color and fontWeight properties.

Reply:
Already moved it in SideBarItemStyled.js